packages/block-library/src/index.js

```diff
@@ -33,6 +33,8 @@ import * as embed from './embed';
 import * as file from './file';
 import * as html from './html';
 import * as mediaText from './media-text';
+import * as navigationMenu from './navigation-menu';
+import * as navigationMenuItem from './navigation-menu/navigation-menu-item';
```

Comment:
`navigationMenuItem` will have to be moved to its own folder to make it work with `block.json`  metadata file. I think it's fine to put it in the root folder.

Reply:
Thanks 👍